testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeSparkIceberg.java

```diff
@@ -57,20 +59,25 @@ public EnvSinglenodeSparkIceberg(Standard standard, Hadoop hadoop, DockerFiles d
     @Override
     public void extendEnvironment(Environment.Builder builder)
     {
+        String dockerImageName = "ghcr.io/trinodb/testing/hdp3.1-hive:" + hadoopImagesVersion;
```

Comment:
looks redundant, perhaps inline it back.

Reply:
This came from `EnvSinglenodeHdp3`. This first parts of the setup here are copied verbatim from there, and I separated it to make it visually similar.

I think one unnecessary variable declaration is fine in exchange for making the code duplication more visible (and hopefully, one day we'll get rid of the duplication entirely and inline the variable in exactly one place).